Sources/Transaction/TransactionBuilder+Math.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+
+extension TransactionBuilder {
+    enum Math {}
+}
+
+extension TransactionBuilder.Math {
+
+    static func totalOutlayCheck(
+        for possibleTransaction: PossibleTransaction,
+        fee: Amount,
+        inputs: [PreparedTxInput]
+    ) -> Bool {
+        let outputs = possibleTransaction.outputs
+        let changeAmount = possibleTransaction.changeAmount
+        let allValues = (outputs.map { $0.amount.value } + [fee.value, changeAmount?.value ?? 0])
+        return UInt64.safeCompare(
+                    sumOfValues: inputs.map { $0.knownTxOut.value },
+                    isEqualToSumOfValues: allValues)
```

Comment:
reduce indentation to double-indent for function params:
```suggestion
                sumOfValues: inputs.map { $0.knownTxOut.value },
                isEqualToSumOfValues: allValues)
```

Reply:
subjective indentation is ok